packages/main/src/TimeSelectionClocks.ts

```diff
@@ -123,6 +140,20 @@ class TimeSelectionClocks extends TimePickerInternals {
 		}
 	}
 
+	/**
+	 * AM/PM segmented button focusin event handler.
+	 */
+	_amPmFocusIn() {
```

Comment:
In TimeSelectionClocks we fire `close-picker` which is not described in the metadata with `@event` decorator.

Reply:
Done.